lib/azure_blob_store.rb

```diff
@@ -79,6 +79,17 @@ def cdn_url(url)
       url.sub("#{schema}#{absolute_base_url}", SiteSetting.azure_cdn_url)
     end
 
+    def url_for(upload, force_download: false)
+      if force_download
+        uri = URI.parse(upload.url)
+        uri.query = [uri.query, "rscd=file;%20attachment"].compact.join('&')
+        url = uri.to_s
```

Comment:
I would recommend using `return uri.to_s` here, then you can eliminate the `else` and have the final line be `'upload.url`

It will save a couple lines of code.

Reply:
Thanks for the tip. :) I'm not used to how the optional `return` statements in Ruby work.